harvester/src/main/java/com/github/reflectoring/infiniboard/harvester/scheduling/SchedulingService.java

```diff
@@ -43,10 +46,10 @@ private JobDataMap createContextData() {
     /**
      * schedules a source update job with its configuration (containing the update time interval)
      */
-    public void scheduleJob(String name, String group, Class<? extends Job> clazz, SourceConfig config) throws SchedulerException {
-        JobDetail job = newJob(clazz).withIdentity(name, group).usingJobData(new JobDataMap(config.getConfigData())).usingJobData(createContextData()).build();
+    public void scheduleJob(String name, String group, Class<? extends Job> clazz, Map<String, String> jobMapDataMap, int interval) throws SchedulerException {
+        JobDetail job = newJob(clazz).withIdentity(name, group).usingJobData(new JobDataMap(jobMapDataMap)).usingJobData(createContextData()).build();
```

Comment:
use less chained calls (e.g. by extracting parts into variables) or format using newlines, if a builder pattern is desired.